modules/mono/glue/GodotSharp/GodotSharp/Core/Extensions/ResourceLoaderExtensions.cs

```diff
@@ -2,7 +2,7 @@ namespace Godot
 {
     public static partial class ResourceLoader
     {
-        public static T Load<T>(string path, string typeHint = null, bool noCache = false) where T : class
+        public static T Load<T>(string path, string typeHint = null, ResourceLoader.CacheMode noCache = (ResourceLoader.CacheMode)1) where T : class
```

Comment:
Not familiar with Mono bindings, but can't this use the actual enum value instead of casting `1`? The whole point of enumd is to avoid hardcoding magic numbers.

Reply:
I copied it from ResourceLoader that was generated by the engine (which mismatch with extension class caused an error). But yeah, probably a good idea to change it to actual enum.